sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -49,8 +49,8 @@ class TranslatorEntraIdAuthenticationPolicy(BearerTokenCredentialPolicy):
     :type credential: ~azure.core.credentials.TokenCredential
```

Comment:
Actually we need to update the docs on the policy as well to include the other properties.

Reply:
So this means resource_id, region, audience

Reply:
OK, added.